views/voting.html

```diff
@@ -43,20 +43,40 @@ <h1><span>Voting Preferences (v{{.VoteVersion}})</span></h1>
 						<label for="accordion-control-{{$i}}">
 							<div class="row accordion__toggle">
 								<div class="col-6">
-									<h2>{{$data.Vote.Id}}</h2>
+									<h2>{{$data.Agenda.Vote.Id}}</h2>
 								</div>
+								{{if ne $data.Status ""}}
+								<div class="col-6 text-right">
+									<!-- labels -->
+									{{if eq $data.Status "upcoming"}}
+									<div class="indicator upcoming">upcoming</div>
+									{{end}}
+									{{if eq $data.Status "in progress"}}
+									<div class="in-progress indicator">in-progress</div>
```

Comment:
Lets be consistent with the hyphens and the caps here. E.G. we currently have "in-progress" and "Locked In".

I think css class names should be lower case and hyphenated, text displayed to the user should be title case with spaces.